ProcessLib/GroundwaterFlow/Tests.cmake

```diff
@@ -706,6 +706,18 @@ AddTest(
     square_1x1_quad_1e3_volumetricsourceterm_analytical_solution.vtu square_1e3_volumetricsourceterm_pcs_0_ts_1_t_1.000000.vtu analytical_solution pressure 1e-10 1e-11
 )
 
+AddTest(
+    NAME GroundWaterFlowProcess_VolumetricSourceTerm_sin_x_sin_y_square_1e3
+    PATH Elliptic/square_1x1_GroundWaterFlow
+    EXECUTABLE ogs
+    EXECUTABLE_ARGS square_1e3_volumetricsourcetermdataarray.prj
+    WRAPPER time
+    TESTER vtkdiff
+    REQUIREMENTS NOT (OGS_USE_LIS OR OGS_USE_MPI)
+    DIFF_DATA
+    square_1x1_quad_1e3_volumetricsourcetermdataarray.vtu square_1e3_volumetricsourcetermdataarray_pcs_0_ts_1_t_1.000000.vtu analytical_solution pressure 2e-2 1e-16
```

Comment:
This compares to an analytical solution? (Maybe sin(x)*sin(y))? Otherwise the absolute tolerance would be pretty bad. Maybe you could mention the fact that you compare to an analytical solution, and also to which one.